x-pack/plugins/observability_solution/dataset_quality/public/services/data_streams_stats/types.ts

```diff
@@ -36,4 +38,7 @@ export interface IDataStreamsStatsClient {
   getDataStreamsEstimatedDataInBytes(
     params: GetDataStreamsEstimatedDataInBytesParams
   ): Promise<GetDataStreamsEstimatedDataInBytesResponse>;
+  getIntegrationDashboards(
```

Comment:
Maybe this one and `getDataStreamDetails` belongs to another service

Reply:
I don't have a strong opinion about this one, just seeing this service growing with every new endpoint and started to think that we might split what is needed to render the page vs what is needed to render the flyout